x-pack/plugins/security_solution/server/lib/app_features/security_assistant_kibana_sub_features.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import type { SubFeatureConfig } from '@kbn/features-plugin/common';
+
+// @ts-expect-error unused variable
+const createConversationSubFeature: SubFeatureConfig = {
```

Comment:
Just trying to understand here: this is a sub-feature of the Assistant that is not needed or used as of now?

Reply:
Ah, just saw your note that this is an example for future reference.

Reply:
Good point though -- I will add the comment at the expect-error as well. I don't normally like leaving un-used code as documentation for future use cases, but in this instance I think it'll be really helpful instead of having to compare against the other two subfeature implementations.